src/Course/List.hs

```diff
@@ -327,11 +310,8 @@ produce f x = x :. produce f (f x)
 -- prop> \x y -> let types = x :: List Int in notReverse x ++ notReverse y == notReverse (y ++ x)
 --
 -- prop> \x -> let types = x :: Int in notReverse (x :. Nil) == x :. Nil
-notReverse ::
-  List a
-  -> List a
-notReverse =
-  error "todo: Is it even possible?"
+notReverse :: List a -> List a
+notReverse = id
```

Comment:
Its a weird exersice, I also put `id`
https://hoogle.haskell.org/?hoogle=%5Ba%5D+-%3E+%5Ba%5D

Reply:
I don't really get why it was there? The tests pass so I assume `id` is correct?